src/makeInfinityAdder.js

```diff
@@ -7,6 +7,23 @@
 
 function makeInfinityAdder() {
   // write code here
+  let result = 0;
+
+  const add = function(number) {
```

Comment:
Since it passed the tests I guess it's ok, but personally I'd either use an arrow function or function declaration